README.md

```diff
@@ -1,6 +1,32 @@
 # shtuff  [![Build Status](https://travis-ci.org/jfly/shtuff.svg?branch=master)](https://travis-ci.org/jfly/shtuff)
 
-Shell stuffer
+Shell stuff will stuff commands into a shell à la `tmux send-keys` or `screen
```

Comment:
I'd prefer to DRY this up. Any ideas? Maybe we ship the README.md in the python package, and our help script can read from the readme.md at runtime?

If that sounds gross, maybe we should add our first test? I could assert that the contents of the readme are identical to the epilogue?

Reply:
I was thinking the same thing when I wrote the readme (I couldn't stand copying/pasting), but I didn't feel like digging in how to strip markdown. I think we're in agreement, though, that updating the help text and readme separately sucks and we need to find a way to clean it up!